.github/workflows/ci.yaml

```diff
@@ -7,18 +7,38 @@ on:
     branches: [ master ]
 
 env:
-  PUBLIC_URL: 'https://klubjagiellonski.github.io/pola-web/'
   # Enable colors in CLI apps.
   # 3 - Truecolor support (16 million colors)
   # https://www.npmjs.com/package/chalk#chalksupportscolor
   FORCE_COLOR: 3
+  AWS_BUCKET_NAME: kj-pola-app-web-preview
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     steps:
+    - name: Set deployment status
+      uses: pbrandone/create-status-action@master
+      if: github.repository == 'KlubJagiellonski/pola-web' && github.event_name == 'pull_request'
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        state: pending
+        description: Preparing deploy preview
+        context: Deploy Preview URL
+    - name: "Set public URL"
+      if: github.repository == 'KlubJagiellonski/pola-web' && github.event_name == 'pull_request'
+      run: |
+        BASE_URL="http://${AWS_BUCKET_NAME}.s3-website.eu-central-1.amazonaws.com"
```

Comment:
```suggestion
        BASE_URL="http://${AWS_BUCKET_NAME}.s3-website.${{secrets.AWS_DEFAULT_REGION}}.amazonaws.com"
```

Reply:
To jest trochę bolesne, ponieważ jest zapisałem to jako sekret.